components/server/src/api/workspace-service-api.ts

```diff
@@ -140,11 +140,9 @@ export class WorkspaceServiceAPI implements ServiceImpl<typeof WorkspaceServiceI
             page.limit,
             page.offset,
         );
-        const resultTotal = results.length;
         const response = new ListWorkspaceSessionsResponse();
         response.workspaceSessions = results.map((session) => this.apiConverter.toWorkspaceSession(session));
-        response.pagination = new PaginationResponse();
-        response.pagination.total = resultTotal;
```

Comment:
This `total` value was misleading, since it just contained the total of the current page, not how many results it's possible to fetch in total.